Packs/PrismaCloud/ReleaseNotes/4_2_24.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Prisma Cloud v2
+
+- Fixed an issue where the arguments *include_resource_json* and *include_additional_resource_fields* were used in the ***prisma-cloud-event-search*** command, instead of the ***prisma-cloud-config-search*** command.
```

Comment:
```suggestion
Fixed an issue where the following arguments were used in the ***prisma-cloud-event-search*** command, instead of the ***prisma-cloud-config-search*** command.
- *include_resource_json*
- *include_additional_resource_fields* 
```